daemon/cluster/errors.go

```diff
@@ -0,0 +1,104 @@
+package cluster
+
+import (
+	"github.com/pkg/errors"
+)
+
+// errNoSwarm is returned on leaving a cluster that was never initialized
+var errNoSwarm = errors.New("This node is not part of a swarm")
+
+// errSwarmExists is returned on initialize or join request for a cluster that has already been activated
+var errSwarmExists = errors.New("This node is already part of a swarm. Use \"docker swarm leave\" to leave this swarm and join another one.")
+
+// errSwarmJoinTimeoutReached is returned when cluster join could not complete before timeout was reached.
+var errSwarmJoinTimeoutReached = errors.New("Timeout was reached before node was joined. The attempt to join the swarm will continue in the background. Use the \"docker info\" command to see the current swarm status of your node.")
```

Comment:
@aaronlehmann Doesn't look like this error was specifically caught by any error checks. Should this remain a http status 500?